static/app/views/replays/detail/network/utils.tsx

```diff
@@ -34,3 +40,44 @@ export function sortNetwork(
     return valueB > valueA ? 1 : -1;
   });
 }
+
+export const getResourceTypes = (networkSpans: NetworkSpan[]) =>
```

Comment:
can we sort this and the status types, so they display in a predictable order for users